dcbr-api/api/views.py

```diff
@@ -2,21 +2,96 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from rest_framework import viewsets
-from api.serializers import CategorySerializer, EntrySerializer
-from api.models import Category, Entry
+from api.serializers import (
+    # CategorySerializer,
+    # EntrySerializer,
+    OperatorSerializer,
+    AddressSerializer,
+    Risk_Factor_Animals_Serializer,
+    Risk_Factor_Operation_Serializer,
+    InspectionSerializer,
+    Association_Membership_Serializer,
+)
+
+# from api.models import Category, Entry, Operator, Address
+from api.models import (
+    Operator,
+    Address,
+    Inspection,
+    Risk_Factor_Animals,
+    Risk_Factor_Operation,
+    Association_Membership,
+)
 
 
 class CategoryViewSet(viewsets.ModelViewSet):
```

Comment:
CategoryViewSet and EntryViewSet should be removed now, as we don't need them anymore.